### PR TITLE
add appwrite  and auth sttings for signup page

### PR DESCRIPTION
**Pull request by chandjamba**
Added appwrite connection integration

### CODE REVIEW

File: src/lib/appwrite/services/auth.service.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  }) {
+    try {
+      return await account.createEmailPasswordSession(email, password);
+    } catch (error) {
+      console.log(error);
+    }
+  },
+  async createUserVerification() {
+    try {
+      return await account.createVerification("url");
```
Comment (kaushalluffa) [inline src/lib/appwrite/services/auth.service.ts:34]:
```suggestion
      // TODO: this url will be replaced after that route is created in frontend
      return await account.createVerification("url");
```

File: src/lib/appwrite/services/auth.service.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    }
+  },
+  async deleteSession() {
+    try {
+      return await account.deleteSessions();
+    } catch (error) {
+      console.log(error);
+    }
+  },
+};
```
Comment (kaushalluffa) [inline src/lib/appwrite/services/auth.service.ts:46]:
```suggestion
async getCurrentUser() {
    try {
      return await account.get();
    } catch (error) {
      console.log(error);
    }
  },
};
```

**Review by kaushalluffa — CHANGES_REQUESTED** [review]

File: src/lib/appwrite/config.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export const APPWRITE_CONFIG = {
+  projectId: import.meta.env.APPWRITE_TWITTER_CLONE_PROJECT_ID,
+  databaseId: import.meta.env.APPWRITE_TWITTER_CLONE_DATABASE_ID,
+};
```
Comment (kaushalluffa) [inline src/lib/appwrite/config.ts:4]:
```suggestion
  projectId: import.meta.env.VITE_APPWRITE_TWITTER_CLONE_PROJECT_ID,
  databaseId: import.meta.env.VITE_APPWRITE_TWITTER_CLONE_DATABASE_ID,
};
```

**Review by kaushalluffa — CHANGES_REQUESTED** [review]

File: src/lib/appwrite/services/auth.service.ts (hunk 2 above)
Comment (chandjamba) [reply src/lib/appwrite/services/auth.service.ts:53]: declared a function to get current user info

File: src/lib/appwrite/config.ts (hunk 3 above)
Comment (chandjamba) [reply src/lib/appwrite/config.ts:4]: fixed wrong pattern

File: src/lib/appwrite/services/auth.service.ts (hunk 1 above)
Comment (chandjamba) [reply src/lib/appwrite/services/auth.service.ts:36]: added

**Review by kaushalluffa — APPROVED** [review]

**Pull request merged**